### ISSUE TITLE
m5copy from etransfer branch does not show version

### ISSUE THREAD

**Issue by varenius** [jive-vlbi/jive5ab#13]
After installing jive5ab from etransfer branch
```
oper@gyller:/usr/local/src/jive5ab.git_etransfer/build$ make install
[  8%] Built target udt5ab
[100%] Built target jive5ab-3.0.0-64bit-Release
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/bin/jive5ab-3.0.0-64bit-Release
-- Installing: /usr/local/bin/DirList.py
-- Installing: /usr/local/bin/SSErase.py
-- Installing: /usr/local/bin/StartJ5
-- Installing: /usr/local/bin/m5copy
```
I wanted to check m5copy version but apparently it is secret:
```
oper@gyller:~$ m5copy  -v
$Id$
```
:)

**Comment by kettenis** [comment]
That happens when you switch from cvs to git...

**Comment by haavee** [comment]
Yes it is **the** major PITA if you ask me. I have not found a proper way to get around this, other than the 'hack' I did by manually inserting `$Id: vX.Y` in the tagged release for e.g. `m5copy` (see README.md). Really open to better suggestions.

**Comment by kettenis** [comment]
The "industry standard" seems to be to report the abbreviated git hash and append a "-dirty" if there are local modifications.  For example:

U-Boot 2021.04-rc3-00180-g65fb72402c-dirty (Mar 14 2021 - 19:02:30 +0100)

The 2021.04-rc3 bit may be based on a release (candidate) tag in this particular example.

**Comment by haavee** [comment]
I think the problem isn't so much as to _what_ to print as version but _how_ to get the code to print that. How can you get a git controlled file to display it's own hash? At least that is the problem I can't figure out the solution to.
